File: Modul 5/Soal 5/PRAK505-2410817210010-AryaArrozzaRidhoSyaputra.py
def Biodata(Namaku, Asal, tahunLahir):
    tahun_sekarang = 2020
    print("\nPerkenalkan nama saya:", Namaku)
    print("Umur Saya:", tahun_sekarang - tahunLahir)
    print("Saya Adalah Angkatan:", tahun_sekarang)
    print("Asal saya dari:", Asal)

def main():
    tahunLahir = int(input())
    Namaku = input()
    Asal = input()
    
    Biodata(Namaku, Asal, tahunLahir)

if __name__ == "__main__":
    main()

File: Modul 5/Soal 5/PRAK505-2410817210010-AryaArrozzaRidhoSyaputra.c
#include <stdio.h>

void Biodata(char Namaku[], char Asal[], int tahunLahir){
    int tahun_sekarang = 2020;
    printf("\nPerkenalkan nama saya: %s\n", Namaku);
    printf("Umur Saya: %d\n", tahun_sekarang - tahunLahir);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal saya dari: %s\n", Asal);
}

int main() {
    int tahunLahir;
    char A[20], B[15];

    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&A);
    scanf(" %[^\n]%*c",&B);
    Biodata(A, B, tahunLahir);

    return 0;
}